src/components/ClientEvents.vue

```diff
@@ -1,6 +1,113 @@
 <template>
   <section class="fields">
-    <h1>Events</h1>
+    <h1>Client Milestones</h1>
+    <div>
+      <div class="columns">
+        <div class="column">
+          <div class="field">
+            <button class="primary" @click="$router.push('/events/add')">
+              Add Event
+            </button>
+          </div>
+
+          <div class="card">
```

Comment:
Fitting cards within a card doesn't look very pretty. Consider dividing different parts with `<section>`.

Reply:
We named the class "Card" however, there are no cards within a card, each is a separate card.